src/darwin/Framework/CHIP/EndpointDescription/MTRAccessGrant.mm

```diff
@@ -0,0 +1,131 @@
+/**
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import "MTRError_Internal.h"
+#import "MTRLogging_Internal.h"
+#import <Matter/MTRAccessGrant.h>
+
+#include <lib/core/CASEAuthTag.h>
+#include <lib/core/GroupId.h>
+#include <lib/core/NodeId.h>
+#include <lib/support/SafeInt.h>
+
+using namespace chip;
+
+@implementation MTRAccessGrant
```

Comment:
I think using `MTR_DIRECT_MEMBERS` on new `@implementations` would be a good habit to get into.

Reply:
Will do.